jwt.js

```diff
@@ -268,7 +294,10 @@ function fastifyJwt (fastify, options, next) {
         }
       },
       function sign (secretOrPrivateKey, callback) {
-        jwt.sign(payload, secretOrPrivateKey, options.sign || options, callback)
+        const signerOptions = mergeOptionsWithKey(options.sign || options, secretOrPrivateKey)
+        const signer = createSigner(signerOptions)
```

Comment:
We should not be creating a signer every time. It's expensive and remove most of the benefits of fast-jwt.